zebra-chain/src/sapling/arbitrary.rs

```diff
@@ -15,24 +20,22 @@ impl Arbitrary for Spend<PerSpendAnchor> {
         (
             any::<tree::Root>(),
             any::<note::Nullifier>(),
-            array::uniform32(any::<u8>()),
+            any::<SpendVerificationKeyBytesWrapper>(),
```

Comment:
I think this just works:

```suggestion
            spendauth_verification_key_bytes(),
```

Reply:
Much better, thanks! Done in 29b13a0